### DISCUSSION TITLE
Find with select issue

### DISCUSSION THREAD

**Discussion by jamesgol** [pods-framework/pods#6836] [Q&A]
I was messing around with some stuff tonight and realized that if I did a find and used the 'select' param it was behaving strangely.

I figured out that if I didn't include `t`.`id` in the fields to be selected then it would only return the first result.

So this either needs to be documented or the code changed to always make sure the id is included

**Comment by sc0ttkclark** [comment]
If you do a custom 'select', you need to include ID or else many things break, we should definitely document that.

**Comment by pglewis** [comment]
Clearing the milestone on this for now, 3.0 culling.
